docs/server/source/server-reference/configuration.md

```diff
@@ -311,7 +311,7 @@ defined by [Python](https://docs.python.org/3.6/library/logging.html#levels),
 but case insensitive for convenience's sake:
 
 ```
-"critical", "error", "warning", "info", "debug", "notset"
+"critical", "error", "warning", "info", "debug", "notset", "benchmark"
```

Comment:
Same comment --- Shouldn't "benchmark" (logging level 15) be between "info" (logging level 20) and logging level "debug" (logging level 10)?